lib/wibox/init.lua

```diff
@@ -253,6 +253,10 @@ local function new(args)
     ret._drawable = wibox.drawable(w.drawable, { wibox = ret },
         "wibox drawable (" .. object.modulename(3) .. ")")
 
+    function ret._drawable.get_wibox()
+        return ret
+    end
```

Comment:
> (And I fail to see how this commit has anything to do with find_widgets even though the commit 
> summary says that the commit is all about that)

Because the clickable or mouse_overable widgets callback information comes from `find_widgets` and if that's used to move a wibox then it has to know the wibox. I agree it's a bit hacky, but fixing that would be a bit complex unless I missed something obvious when I wrote that (didn't look recently). Feel free to propose a better solution, but I don't have much AwesomeWM time and a backlog of > 500 commits to finish, so it's a bit hopeless to expect me working on this in any kind of timely fashion.